static/main.css

```diff
@@ -28,6 +28,14 @@
     cursor: pointer;
 }
 
+.row.hover-row.build-table.clickable.collapsed {
```

Comment:
Try not to reflect exactly how HTML looks like. Be more liberal.
This should be enough:
```css
.build-table {
    background: #ccc;
}
.build-table.collapsed {
    background: #fff;
}
```

Reply:
Changes to .build-table.clickable { and .build-table.clickable.collapsed { because .build-table refer also to top "zero" row with column names. Other words adding class .clickable.